app/src/ui/components/Header.module.css

```diff
@@ -31,9 +31,9 @@ h2 {
 }
 
 .color-yellow {
-  color: var(--yellow);
+  composes: typography-secondary from global;
```

Comment:
`.color-red {
  color: var(--red);
}

.color-blue {
  color: var(--blue);
}` - those colors does not exist anymore. please remove it or adjust the theme with this colors